frontend/src/main.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App.tsx';
-import './index.css';
 
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+ReactDOM.createRoot(document.getElementById('root')!).render(
```

Comment:
eslint reports `Forbidden non-null assertion  @typescript-eslint/no-non-null-assertion` here

Reply:
thanks for reminding me, fixed